flocker/control/test/persisted_model.json

```diff
@@ -149,6 +152,10 @@
                     "flocker.control._model.Node"
                 ]
             },
+            "flocker.control._model.PersistentState": {
+                "category": "record",
+                "fields": {}
```

Comment:
No action required: Much more clear with the comments in `_model`. If I ever make an arbitrary decision that I need to be persisted (and is not configuration), then it goes in this bucket.